### PR TITLE
[ADD] udes_warehouse_classification: Add product classification string

### PR DESCRIPTION
**Pull request by williamsajr**
User-story: 10344
Signed-off-by: Anthony Williams <anthony.williams@unipart.io>

### CODE REVIEW

File: addons/udes_warehouse_classification/models/product_product.py [hunk 1]
```diff
@@ -7,3 +7,12 @@ class ProductProduct(models.Model):
     def get_classification_messages_for_report(self, report_name):
         """Find product classification messages needed for a given report"""
         return self.mapped("product_tmpl_id").get_classification_messages_for_report(report_name)
+
+    def get_printable_classification_names(self, seperator="", order_key="name"):
```
Comment (ws-mps) [inline addons/udes_warehouse_classification/models/product_product.py:11]: Typo: separator ?

Comment (williamsajr) [reply addons/udes_warehouse_classification/models/product_product.py]: Ooops, fixed

File: addons/udes_warehouse_classification/models/product_product.py [hunk 2]
```diff
@@ -7,3 +7,12 @@ class ProductProduct(models.Model):
     def get_classification_messages_for_report(self, report_name):
         """Find product classification messages needed for a given report"""
         return self.mapped("product_tmpl_id").get_classification_messages_for_report(report_name)
+
+    def get_printable_classification_names(self, separator="", order_key="name"):
+        """Return a string of the prodcut classifications in self"""
```
Comment (squah-u) [inline addons/udes_warehouse_classification/models/product_product.py:12]: typo: `product`

Comment (williamsajr) [reply addons/udes_warehouse_classification/models/product_product.py]: Fixed

**Review by ws-mps — APPROVED** [review]

**Review by kenvac — APPROVED** [review]

**Pull request merged**